test-utils/README.md

```diff
@@ -1 +1,73 @@
-## Test Utils
+# Test Utils
+
+## Testing from a background thread to let `Dispatchers.Main` function properly
+
+By default Kotlin/Native tests run on the UI thread, similar to regular Swift tests.
+
+As with these regular test, this presents problems when trying to dispatch work to the main thread.
+In particular for Kotlin bases test, usinf the `Dispatchers.Main`dispatcher -either directly or indirectly- will not work.
```

Comment:
typo in using

Reply:
that whole sentence was garbled 9d607ff1c70d7789ca7b1871e3ffe9b6f67ca8a0